src/buildstream/element.py

```diff
@@ -2810,14 +2807,11 @@ def __sandbox(self, directory, stdout=None, stderr=None, config=None, allow_remo
         else:
             output_node_properties = None
 
-        if directory is not None and allow_remote and context.remote_execution_specs:
-
-            self.info("Using a remote sandbox for artifact {} with directory '{}'".format(self.name, directory))
-
+        if allow_remote and context.remote_execution_specs:
+            self.info("Using a remote sandbox for artifact {}".format(self.name))
```

Comment:
Only thing I'm not sure is whether we want to keep this message at all. I think it's pretty clear from the other messages that we're doing a remote build.

Reply:
I don't regularly do remote builds, is it clear ?

I would be happy if the only difference was a clear indication in the startup summary.

We can surly remove this info message.

Reply:
Yeah, it's very clear. You get "uploading build root" and "waiting for the remote build to complete" as subtasks of "running commands".